packages/Vendor/tests/Unit/Services/VendorStatusTransitionPolicyTest.php

```
@@ -0,0 +1,79 @@
+                VendorStatus::Draft,
+                'Cannot transition vendor status from Archived to Draft.',
+            ],
+        ];
+    }
+
+    #[DataProvider('allowedTransitionsProvider')]
+    public function test_allowed_transitions_do_not_throw(VendorStatus $from, VendorStatus $to): void
+    {
+        $this->policy->assertCanTransition($from, $to);
+        self::addToAssertionCount(1);
+    }
```

PHPMD (2.15.0)

[error] The method test_allowed_transitions_do_not_throw is not named in camelCase. (undefined)

(CamelCaseMethodName)